### PR TITLE
Feature cell in file

### PR DESCRIPTION
**Pull request by BrandonKroes**
Cell in File has been finished and is ready for the merge to release

### CODE REVIEW

File: module/Cell.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        if self.get_x_y() != item.get_x_y():
+            self.previous = item
+
+    def __getattr__(self, key):
+        return object.__getattribute__(self, key)
+
+    def get_x_y(self):
+        """ Setting the previous Cell.
+                Detects if the item points to itself.
+
```
Comment (nicovanbentum) [inline module/Cell.py]: wrong doc I think

File: module/Cell.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        Afterwards removing the file_name, conserve memory and can be generated.
+        """
+
+        with open(generate_file_name(self.x, self.y), 'r') as f:
+            for key, values in json.load(f).items():
+                setattr(self, key, values)
+        delattr(self, 'file_name')
+
+    def set_file_content(self):
+        """ Getting the content from a cell in file object.
```
Comment (nicovanbentum) [inline module/Cell.py]: Think you meant "setting"

File: module/Cell.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        Using the native json.dump function to store it in a JSON string.
+        Uses the __dict__ call to store ALL object attributes
+        """
+        with open(generate_file_name(self.x, self.y), 'w') as f:
+            json.dump(self.__dict__, f)
+
+    def __delete__(self, instance):
+        self.set_file_content()
```
Comment (nicovanbentum) [inline module/Cell.py]: does this also delete the file from disk?

File: module/Cell.py (hunk 2 above)
Comment (BrandonKroes) [reply module/Cell.py]: Fixed

File: module/Cell.py (hunk 1 above)
Comment (BrandonKroes) [reply module/Cell.py]: Indeed, fixed.

File: module/Cell.py (hunk 3 above)
Comment (BrandonKroes) [reply module/Cell.py]: No we want to let the file remain on the disk, we only want to clear the memory it contains.

**Pull request merged**